lib/hyacinth/adapters/external_identifier_adapter/datacite.rb

```diff
@@ -64,6 +64,17 @@ def update_impl(id, digital_object, target_url, state)
           true
         end
 
+        def deactivate_impl(id)
+          json_payload = datacite_payloads.build_state_update(:inactive)
```

Comment:
Should this be `:registered` instead of `:inactive`?  ping @fcd1

Reply:
it should be, and it should also probably not set `:draft` DOIs to `:registered` here.